docs/core/diagnostics/distributed-tracing-instrumentation-walkthroughs.md

```diff
@@ -0,0 +1,412 @@
+---
+title: Add distributed tracing instrumentation - .NET
```

Comment:
>title [](start = 0, length = 5)

as we removed some of the contents from the distributed-tracing doc, I think we need to cover some more information in this doc (e.g. ActivityLinks and ActivityKind).

Reply:
Added